README.md

```diff
@@ -1,2 +1,6 @@
 # app
```

Comment:
Could you extend this README with more information? For example:
- What is the purpose of this model service
- What is the main entrypoint and how to start up the app
- What are its main components (and possibly how they connect to each other)

Reply:
Ok I see there is another README with instructions - can you merge these 2 into 1 and add the missing points above? ;)

Reply:
Ok, changed it. Please check again.